content/configuration/devtool.md

```diff
@@ -26,53 +26,84 @@ Choose a style of [source mapping](http://blog.teamtreehouse.com/introduction-so
 
 T> The webpack repository contains an [example showing the effect of all `devtool` variants](https://github.com/webpack/webpack/tree/master/examples/source-map). Those examples will likely help you to understand the differences.
 
-devtool                       | build | rebuild | production | quality
------------------------------ | ----- | ------- | ---------- | -----------------------------
-eval                          | +++   | +++     | no         | generated code
-cheap-eval-source-map         | +     | ++      | no         | transformed code (lines only)
-cheap-source-map              | +     | o       | yes        | transformed code (lines only)
-cheap-module-eval-source-map  | o     | ++      | no         | original source (lines only)
-cheap-module-source-map       | o     | -       | yes        | original source (lines only)
-eval-source-map               | --    | +       | no         | original source
-source-map                    | --    | --      | yes        | original source
-inline-source-map             | --    | --      | no         | original source
-hidden-source-map             | --    | --      | yes        | original source
-nosources-source-map          | --    | --      | yes        | without source content
-
-T> `+` means faster, `-` slower and `o` about the same time
-
-Some of these values are suited for development and some for production. For development you typically want fast Source Maps at the cost of bundle size, but for production you want separate Source Maps that are accurate.
+T> Instead of using the `devtool` option you can also use the `SourceMapDevToolPlugin`/`EvalSourceMapDevToolPlugin` directly. It has more options. You must never use both `devtool` option and plugin!
```

Comment:
T> Instead of using the `devtool` option you can also use `SourceMapDevToolPlugin`/`EvalSourceMapDevToolPlugin` directly as it has more options. Never use both the `devtool` option and plugin together.